engine/lite/embeddedDeploymentManager/src/main/scala/pl/touk/nussknacker/streaming/embedded/EmbeddedDeploymentManager.scala

```diff
@@ -52,34 +53,47 @@ object EmbeddedDeploymentManager extends LazyLogging {
   checking status, but for this @volatile on interpreters should suffice.
  */
 class EmbeddedDeploymentManager(modelData: ModelData, engineConfig: Config,
+                                processingTypeDeploymentService: ProcessingTypeDeploymentService,
                                 handleUnexpectedError: (ProcessVersion, Throwable) => Unit)(implicit ec: ExecutionContext) extends BaseDeploymentManager with LazyLogging {
 
   private val metricRegistry = LiteEngineMetrics.prepareRegistry(engineConfig)
 
   private val contextPreparer = new LiteEngineRuntimeContextPreparer(new DropwizardMetricsProviderFactory(metricRegistry))
 
-  @volatile private var interpreters = Map[ProcessName, ScenarioInterpretationData]()
+  @volatile private var interpreters: Map[ProcessName, ScenarioInterpretationData] = {
+    val deployedScenarios = Await.result(processingTypeDeploymentService.getDeployedScenarios, 10.seconds)
```

Comment:
let's extract `10.seconds` to some constant... so it's more visible. OTOH, I wonder if this method will ever be invoked without await ;)

Reply:
constant extracted + rebased to staging

Reply:
:+1: